src/components/classifier/QuestionClassifier.js

```diff
@@ -43,215 +45,229 @@ class QuestionClassifier extends Component {
+                                    type="answer"
+                                    text="Submit"
+                                    onPress={this.submitClassification()}
+                                />
+                            </View>
+                            {
+                                (task.help || R.length(guide.items) > 0) &&
+                                <Separator style={styles.separator}/>
+                            }
+                            {
+                                task.help !== null &&
+                                <NeedHelpButton
+                                    onPress={() => this.classifierContainer.displayHelpModal()}
+                                />
+                            }
+                            {
+                                R.length(guide.items) > 0 &&
+                                <ClassifierButton
+                                    onPress={() => this.classifierContainer.displayFieldGuide()}
+                                    style={styles.guideButton}
+                                    text="Field Guide"
+                                    type="guide"
+                                />
+                            }
+                        </ScrollView>
+                        :
+                        renderTutorial()
+                }
+                <FullScreenImage
+                    source={fullScreenImageSource}
+                    isVisible={showFullSize}
+                    handlePress={() => this.setState({fullScreenQuestion: '', showFullSize: false})}
+                    question={fullScreenQuestion}
+                />
+            </View>
 
         return (
-                <View style={styles.container}>
-                    <ClassifierContainer
-                        inBetaMode={inBetaMode}
-                        project={project}
-                        help={task.help}
-                        guide={guide}
-                        ref={ref => this.classifierContainer = ref}
-                    >
-                        { needsTutorial ? renderTutorial() : classificationPanel }
-                    </ClassifierContainer>
-                </View>
+            <View style={styles.container}>
+                <ClassifierContainer
+                    inBetaMode={inBetaMode}
+                    project={project}
+                    help={task.help}
+                    guide={guide}
+                    ref={ref => this.classifierContainer = ref}
+                >
+                    {needsTutorial ? renderTutorial() : classificationPanel}
+                </ClassifierContainer>
+            </View>
         );
     }
 }
 
 QuestionClassifier.propTypes = {
-    project: PropTypes.object,
     workflow: PropTypes.object,
-    display_name: PropTypes.string,
+    project: PropTypes.shape({
+        id: PropTypes.number,
+        display_name: PropTypes.string,
+        in_museum_mode: PropTypes.bool,
```

Comment:
Same note on making this camel case, unless this is returned from the api in snake case.